structure/structure.py

```diff
@@ -15,24 +15,52 @@
 from griptape.tools import Calculator
 
 
-# Returns True if this program is being run from within Griptape Cloud or via the Skatepark emulator
-def is_running_in_managed_environment():
+def is_running_in_managed_environment() -> bool:
+    """Determine if the program is running in a managed environment (e.g., Griptape Cloud or Skatepark emulator).
+    Returns:
+        bool: True if the program is running in a managed environment, False otherwise.
+    """
     return "GT_CLOUD_RUN_ID" in os.environ
 
 
+def get_listener_url() -> str:
+    """The event driver requires a URL, stored in the GT_CLOUD_BASE_URL 
+    environment variable.
+    In Griptape Cloud, this environment variable is provided for you.
+    In Skatepark, you may optionally provide your own if you have overridden
+    the default URL and port that the emulator is listening on. Otherwise,
+    it will use the default URL and port.
+
+    Returns:
+        The URL of the host listening to the Structure.
+    """
+    return os.environ.get("GT_CLOUD_BASE_URL")
```

Comment:
The framework will do this for the user, not needed.

Reply:
a) I wrote this before I saw your framework change
b) where's a good place to tell the user about it?

Reply:
b) I added the info in the `GriptapeCloudEventListenerDriver` class docstring but that still may not be enough. Maybe just a comment near the Driver instantiation?

Reply:
@SavagePencil can we delete this now?